<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gitorious::Internal::GitoriousProjectWidget</class>
 <widget class="QWidget" name="Gitorious::Internal::GitoriousProjectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTreeView" name="projectTreeView"/>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="infoToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttonVerticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="updateCheckBox">
     <property name="text">
      <string>Keep updating</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="Utils::FilterLineEdit" name="filterLineEdit"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::FancyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/fancylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Utils::FilterLineEdit</class>
   <extends>Utils::FancyLineEdit</extends>
   <header location="global">utils/filterlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
